kube-janitor/templates/cron-job.yaml

```diff
@@ -27,6 +27,10 @@ spec:
               - "{{ .Values.kubejanitor.namespace }}"
               - "-annotation"
               - "{{ .Values.kubejanitor.annotation }}"
+              {{- if .Values.kubejanitor.pendingJobExpiration }}
+              - "-pendingJobExpiration"
```

Comment:
Legitimate question: I know eventually we'll cover pod expiration, would it be better to have `-pendingExpiration` rather than have `-pendingJobExpiration` and `-pendingPobExpiration`.

Reply:
If you think that people will always have the same timeout for both pending jobs and pods, then I think I would agree with you. But I think I could see a use case where you may want the timeouts to be different, or only have one turned on at a time.